cookbook/specs/spec_add_manta_pacific.json

```diff
@@ -0,0 +1,118 @@
+{
+    "proposal": {
+        "title": "Add Specs: Manta Pacific",
+        "description": "Adding new specification support for relaying Manta Pacific data on Lava",
+        "specs": [
+            {
+                "index": "MANTAPACIFIC",
+                "name": "manta pacific mainnet",
+                "enabled": true,
+                "imports": [
+                    "ETH1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 1,
+                "blocks_in_finalization_proof": 3,
+                "average_block_time": 10000,
+                "allowed_block_lag_for_qos_sync": 2,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "47500000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "jsonrpc",
+                            "internal_path": "",
+                            "type": "POST",
+                            "add_on": ""
+                        },
+                        "apis": [],
+                        "headers": [],
+                        "inheritance_apis": [],
+                        "parse_directives": [],
+                        "verifications": [
+                            {
+                                "name": "chain-id",
+                                "values": [
+                                    {
+                                        "expected_value": "0xa9"
+                                    }
+                                ]
+                            },
+                            {
+                                "name": "pruning",
+                                "values": [
+                                    {
+                                        "latest_distance": 1990000
```

Comment:
will require nodes to have 1,990,000 blocks to operate, quite a lot, and every request of bigger than 17040 will go to archive, rule of thumb is 24h of blocks, with 10s per block it's 24*3600/10 blocks

Reply:
Ah, sorry about that. I misunderstood that value. I thought it was the block height threshold for archive vs full, but I understand now. I'll get this changed and pushed!